test/rai/boston_analyse_config/pdf_gen.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
How is this file specified?

Reply:
linked in instruction md for score card creation.